common/shlibs

```diff
@@ -1028,6 +1028,7 @@ libuchardet.so.0 uchardet-0.0.6_1
 libgtksourceviewmm-3.0.so.0 gtksourceviewmm-3.2.0_1
 libyajl.so.2 yajl-2.0.1_1
 libconfuse.so.2 confuse-3.2.1_1
+liblldb.so glibc-2.29_1
```

Comment:
This is incorrect. The glibc package contains no such file and only proper SONAMES should be added to this file.

Reply:
I'm sorry if I missed something here, I've read through the documentation and also did a file search on my system for the `liblldb.so` and couldn't find it...

Also, without the `liblldb.so glibc-2.29_1` in the `common/shlibs`, the package won't even build.

I've also tried adding `liblldb.so intellij-idea-community-edition-2020.1.1_1` to `common/shlibs`, and it builds successfully but the library is ignored, even though the package works flawlessly.

How should I solve this? I'm sorry for my lack of experience in this packaging process :/

Reply:
That change could break other packages since `glibc` doesn't contain `lldb.so`. The file that depends on `liblldb.so` in the package is `lib/intellij-idea/plugins/Kotlin/bin/linux/LLDBFrontend`. 

This may need to be added to `skiprdeps`.

Reply:
Ok thanks, hopefully it's now fixed, thanks for the guideance :+1: